packages/yarnpkg-core/sources/Cache.ts

```diff
@@ -124,11 +124,8 @@ export class Cache {
       await xfs.mkdirPromise(this.cwd, {recursive: true});
 
       const gitignorePath = ppath.resolve(this.cwd, `.gitignore` as Filename);
-      const gitignoreExists = await xfs.existsPromise(gitignorePath);
 
-      if (!gitignoreExists) {
-        await xfs.writeFilePromise(gitignorePath, `/.gitignore\n*.lock\n`);
-      }
+      await xfs.changeFilePromise(gitignorePath, `/.gitignore\n*.lock\n*.flock\n`);
```

Comment:
I don't think we use `.lock` anymore, we can remove it
```suggestion
      await xfs.changeFilePromise(gitignorePath, `/.gitignore\n*.flock\n`);
```

Reply:
Only files with `.lock` we have would be `yarn.lock` and I don't see us writing those in there :thinking: